src/software/pipeline/main_panoramaCompositing.cpp

```diff
@@ -1093,7 +2170,8 @@ int aliceVision_main(int argc, char **argv)
   po::options_description optionalParams("Optional parameters");
   optionalParams.add_options()
     ("compositerType,c", po::value<std::string>(&compositerType)->required(), "Compositer Type [replace, alpha, multiband].")
-    ("overlayType,c", po::value<std::string>(&overlayType)->required(), "Overlay Type [none, borders, seams].");
+    ("overlayType,c", po::value<std::string>(&overlayType)->required(), "Overlay Type [none, borders, seams].")
```

Comment:
Not really related to the PR, but it would be good to add an option to combine "borders" and "seams" in a single run (with different colors). We could add a mode "all" for that.

Reply:
Will do